docs/IO.md

```diff
@@ -43,7 +43,10 @@ Starting from a BIDS compliant dataset, this command creates:
 1. A TSV file for each session available with the list of the modalities found for each subject.
 The name of the files produced will be `<prefix>_ses-<session_label>.tsv`.
 2. A text file containing the number and the percentage of modalities missing for each session.
-The name of the files produced will be `<prefix>_summary.txt`.
+The name of the file produced will be `<prefix>_summary.txt`.
```

Comment:
nitpicking but maybe start directly with the name of the file (e.g <prefix>summary.txt: A text file containing the [...])

Reply:
Done in https://github.com/aramis-lab/clinica/pull/272/commits/6956e994ee02fb6da032f0c011243cd905b4dd3f